lib/gui/app/components/flash-results/flash-results.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// eslint-disable-next-line no-unused-vars
+import * as _ from 'lodash';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import styled from 'styled-components';
+import { left, position, space, top } from 'styled-system';
+import { Underline } from '../../styled-components';
+
+const Div: any = styled.div`
+  ${position}
+  ${top}
+  ${left}
+  ${space}
+`;
+
+/* eslint-disable no-inline-comments */
+
+const FlashResults: any = ({ errors, results, message }: any) => {
```

Comment:
No `any` please.
Can we use the `function FlashResults({ errors, results, message }) {` form ?